[TRANSLATION] Russian translation update

A-Rafael opened the pull request.

Fraizeraust commented:
Hello and thank you for your interest on ReactOS!

First and foremost, the contributors [must use their full real name and real E-mail address](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations) for commits. Make sure you [amend your commit with your name & E-mail address](https://stackoverflow.com/questions/3042437/change-commit-author-at-one-specific-commit/3042512#3042512) and update your PR.

A-Rafael commented:
@Fraizeraust, Hello, [commit](https://github.com/A-Rafael/reactos/commit/b5d13e07c39a5b0456b3f68789af9fe41f87e6aa) has a name and email, I updated the github profile and added a name to the profile.
